src/bitcoin-tx.cpp

```diff
@@ -767,7 +767,7 @@ static std::string readStdin()
     if (ferror(stdin))
         throw std::runtime_error("error reading stdin");
 
-    boost::algorithm::trim_right(ret);
+    ret = TrimString(ret, " \f\n\r\t\v");
```

Comment:
this changes functionality (trim right to general trim), I don't think you should do that at the same time

Reply:
@laanwj We don't have any right-trim function and from what I can see there is no harm in also left-trimming in this case. If you prefer I can add a right-trim function, should I do that or is it okay to left-trim?

Reply:
@jgarzik You added this function in fb14452c6c - do you think it is OK to trim (instead of only right-trimming) this string in order to avoid having to add a project local right-trim? :-)